src/main/java/Duke/Deadline.java

```diff
@@ -0,0 +1,28 @@
+package Duke;
+
+class Deadline extends Task {
+    String description;
+    String by;
+    Deadline(String description, String by) {
+        this.description = description;
+        this.by = by;
+    }
+
+    @Override
+    void initializeMessage() {
+        System.out.println("Got it! I've added this task:");
+        displayMessage();
+        System.out.println("Now you have "+ TaskManager.noOfTasks +" tasks in the list.");

```

Comment:
As per the Java coding standard it is recommended to have white spaces before and after your "+" operator.